.github/workflows/auto-review-bot.yml

```diff
@@ -22,30 +22,13 @@ jobs:
         id: save-pr-number
         run: echo "::set-output name=pr::$(cat pr-number.txt)"
 
-      - name: Checkout
-        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-        with:
-          repository: ethereum/EIPs # Default, but best to be explicit here
-          ref: master
-
-      - name: Setup Node.js Environment
-        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93
-        with:
-          node-version: 16
-
       - name: Auto Review Bot
         id: auto-review-bot
-        uses: ethereum/EIP-Bot@b81356bd8302a99e2d2bcf2bb5d2d983a12f7b8d
+        uses: Pandapip1/eip-review-bot@dist
```

Comment:
I love this change. Just a nitpick: would you move it to the `ethereum` or `etheruem-cat-herders` org so it can be a community repo instead of a personal repo?

Reply:
Couldn't agree more. I can't do that without the help of someone with the permission to create repos in the ethereum org (a perm that I don't have), however.

Reply:
How about putting it into the ECH? I think pooja probably have permission.

Reply:
I would be okay with that @poojaranjan

Reply:
I agree - but they can always also then be moved to ethereum afterward. I'll ask if @poojaranjan has create repo perms on @ethereum.